### ISSUE TITLE
Extension suggests only absolute path to file

### ISSUE THREAD

**Issue by buczekmatthias** [nhedger/vscode-inertia#30]
Hello,

for a while I have this problem that extension only suggests absolute path to file instead of `File` or `Folder/File` way as it was intended to do.

I didn't change any default settings of extension.

![ext](https://github.com/nhedger/vscode-inertia/assets/41841715/9ab749c7-5233-48d9-a605-5a6f14c88cde)

**Comment by nhedger** [comment]
Hey @buczekmatthias,

Would you be able to provide a reproduction repository?

If you have changed the default settings in the meantime, please also include a copy of those

**Comment by buczekmatthias** [comment]
Thanks for response @nhedger,

problem appears in all repositories, even fresh ones, [here](https://github.com/buczekmatthias/ext-test) is fresh repo as example.

I didn't make any changes in settings.

**Comment by nhedger** [comment]
Weird, I can't seem to reproduce.

https://github.com/nhedger/vscode-inertia/assets/649677/eb6af6ab-731e-481e-8d47-0aa6008fd547

Seems to work on macOS at least, looks like you're using Windows, I'll try to reproduce on a windows machine when I have a minute.

**Comment by nhedger** [comment]
Can you confirm that your `inertia.pages` setting is set to `resources/js/Pages/**/*.vue` ?

**Comment by buczekmatthias** [comment]
Yes it is. I just tested setting this setting to that value. Also made tests with no extensions and default settings of VSCode and other extensions, it gave same result.

**Comment by nhedger** [comment]
Alrighty, thanks for the info, I'll let you know asap.

**Comment by nhedger** [comment]
Ok so it looks like you'll want to change `resources/js/Pages/**/*.vue` to `resources\js\Pages\**\*.vue` on windows. Could you try and let me know how it goes ?

**Comment by buczekmatthias** [comment]
Unfortunately, it didn't work for me, suggestion is still absolute path. Tried once again with clean VSC profile to ensure it's not messed by some setting or extension.

**Comment by nhedger** [comment]
Kinda hard to reproduce. I'll leave this open just in case someone encounters a similar issue and has more insight. If you happen to find out what is happening, please let me know.

**Comment by CarlosEGuerraSilva** [comment]
Same issue here
![image](https://github.com/user-attachments/assets/45c44015-f32a-4a81-b475-14be8bb9f0b0)
And when you autocomplete inserts full path

**Comment by zhaolinlau** [comment]
@nhedger @buczekmatthias @CarlosEGuerraSilva 

I've just tested, and the conclusion is as below:


When I open the project through the terminal using `code .`, then absolute paths(`c:/xampp/hrdocs/.../Tasks/Index`) will be suggested in auto completion. However, it works fine(`Tasks/Index`) if I open the project through vscode.

**Comment by nhedger** [comment]
Thanks I'll give it a try !

**Comment by nhedger** [comment]
I was able to reproduce with your instructions @zhaolinlau, thanks!

**Comment by nhedger** [comment]
Aiming for later today

**Comment by nhedger** [comment]
v1.0.0 released
